test/tsconfig.json

```diff
@@ -5,6 +5,6 @@
     "noImplicitAny": false
   },
   "include": [
-    "./**/*.ts"
+    "./**/*.ts", "fixtures/resource-limits.js"
```

Comment:
Is this necessary? TS doesn’t do anything for `.js` flles, I think

Reply:
what the heck? I think this was vscode trying to be "helpful"